src/allmydata/stats.py

```diff
@@ -170,7 +174,18 @@ def get_stats(self):
         return ret
 
     def remote_get_stats(self):
-        return self.get_stats()
+        # The remote API expects keys to be bytes:
```

Comment:
Is there anything that invokes `remote_get_stats()`?  I have not been able to find a single use of this from just grepping.

Reply:
I think the remote_* methods get invoked via foolscap, so there's no direct API call but foolscap does a `getattr("remote_" + name)` somewhere.